tests/compiler/std/string.ts

```diff
@@ -1,6 +1,17 @@
 // preliminary
 
 var str: string = "hi, I'm a string";
+var str123: string = "123";
+var str123ab: string = "123ab";
+var strab123: string = "ab123";
+var strHex: string = "0x10";
+var strHexInvalid: string = "0x";
+var strHexnInvalid: string = "-0x";
+var strHexnnInv: string = "--0x10";
+// var strHexInvChar: string = "0x1g";
```

Comment:
In this case we get unreachable

Reply:
Hmm, that's strange. Probably hitting an assertion somewhere then. Might make sense to hook `assert` up to an import on failure now, so we know where that is :)

Reply:
It always print:
```
Testing compiler/std/string.ts
parse incl. I/O: 4.978ms / compile: 4.763ms
validate OK
[trap unreachable]
interpret ERROR:undefined
Created
Created optimized
```

And can't figure out what is problem